packages/itwin/ec3-widget/api/ec3-widget-react.api.md

```diff
@@ -0,0 +1,93 @@
+## API Report File for "@itwin/ec3-widget-react"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="react" />
+
+import type { AccessToken } from '@itwin/core-bentley';
+import type { EC3ConfigurationLabel } from '@itwin/insights-client';
+import { default as React_2 } from 'react';
+import { StagePanelLocation } from '@itwin/appui-react';
+import { StagePanelSection } from '@itwin/appui-react';
+import type { UiItemsProvider } from '@itwin/appui-react';
+import type { Widget } from '@itwin/appui-react';
+
+// @public
+export interface Configuration {
+    // (undocumented)
+    description: string;
+    // (undocumented)
+    displayName: string;
+    // (undocumented)
+    id?: string;
+    // (undocumented)
+    labels: EC3ConfigurationLabel[];
+    // (undocumented)
+    reportId?: string;
+}
+
+// @beta
+export type EC3ConfigProps = EC3ConfigPropsWithRedirectUri | EC3ConfigPropsWithGetEC3AccessToken;
```

Comment:
It looks like neither `EC3ConfigPropsWithRedirectUri` nor `EC3ConfigPropsWithGetEC3AccessToken` are exported.

Reply:
Fixed